macros/edr/data_monitoring/anomaly_detection/get_anomaly_scores_query.sql

```diff
@@ -104,6 +108,8 @@
                 {{ elementary.current_timestamp_column() }} as detected_at,
                 full_table_name,
                 column_name,
+                dimension,
+                dimension_value,
```

Comment:
Is it ok that the columns are not at the end here? 
We don't merge these tables to a big incremental table?